src/jvmMain/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvFileWriter.kt

```diff
@@ -22,6 +22,7 @@ class CsvFileWriter internal constructor(
      */
     private val stateHandler = CsvWriterStateHandler()
 
+    private val quoteNeededChars = setOf('\r', '\n', ctx.quote.char, ctx.delimiter)
```

Comment:
nit:
`quoteNeededChars` is only used on `attachQuote` method, so declare it inside the method.

Reply:
It doesn't seem very efficient to me to create a new set every time `attachQuote()` is called, given that the contents of the set don't change. Is this intended?

Reply:
I understand your point. I agree with your! 👍